frontend/src/app/poll/poll.module.ts

```diff
@@ -48,6 +49,7 @@ import {ChooseDateService} from './services/choose-date.service';
   ],
   providers: [
     ChooseDateService,
+    PollService,
```

Comment:
Ist der nicht providedIn: 'root'? Warum jetzt auf einmal hier aufgetaucht?

Reply:
Whoops, missed that annotation.